code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -826,6 +826,21 @@
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
 	real = FALSE
 
+/obj/item/clothing/suit/hooded/wintercoat/durathread
+	name = "durathread winter coat"
+	desc = "The one coat to rule them all. Extremely durable while providing the utmost comfort."
+	icon_state = "coatdurathread"
+	item_state = "coatdurathread"
+	armor = list("melee" = 40, "bullet" = 25, "laser" = 25, "energy" = 25, "bomb" = 50, "bio" = 50, "rad" = 50, "fire" = 50, "acid" = 50)
```

Comment:
Armor is a tad too strong, it should be an edge weaker than the durathread vest, here values for reference:
https://github.com/Citadel-Station-13/Citadel-Station-13/blob/201a2056b95298164b61d09aceaf73682daa93f7/code/modules/clothing/suits/armor.dm#L250